_episodes/14-runtime.md

```diff
@@ -4,8 +4,11 @@ teaching: 10
 exercises: 0
 questions:
 - "How do I create required input files from input parameters at runtime?"
+- "How do I invoke a script rather than just a simple command line?"
+- "How do I make inputs available to my script?"
 objectives:
 - "Learn how to create files on the fly during runtime."
+- "Learn how to use expressions in bash scripts."
```

Comment:
Each lesson should introduce a single concept and take about 15 minute to read; this might be too much for this lesson..

Reply:
I think this is a small enough package. 

In the original lesson we have a shell expression and how to inject into it using a CWL expression.

But I've never seen an actual user want to do just this. Usually they are asking the question "How do I get this set of commands to work. I know, I'll use a bash script!" and then they try to do it in CWL using file create, and go away frustrated because CWL messes up the `$`.  

So this lesson teaches - as one unit - what you are likely to be looking for when learning that you can create files where you inject variables.

Reply:
I think it's important to show that you need to be vary of `$` - I tried to express it as a **Hint**